README.md

```diff
@@ -85,14 +85,18 @@ python -m goth start your/output/dir/goth-config.yml
 
 If everything went well you should see the following output:
 ```
-Now run your requestor agent as follows:
+Local goth network ready!
 
-$ PATH=/tmp/...:$PATH YAGNA_APPKEY=3438...7901 YAGNA_API_URL=http://172.19.0.6:6000 GSB_URL=tcp://172.19.0.6:6010 examples/blender/blender.py --subnet goth
+You can now load the requestor configuration variables to your shell:
```

Comment:
Why not leave the old way of starting the requestor scripts as an alternative? I'm quite used to it, I always just copy the line with the environment vars and replace the script path (and now I wouldn't have to add `--subnet goth`, cool!). In contrast to `source`'ing the `.env` file, the "old way" does not modify my current environment.

Reply:
Added this back in babcea9